Green reset failed: add troubleshooter (#610)
NabuCasa/support: src/green/troubleshooting/green-reset-failed-what-to-do.md

---
zendesk:
  article_id: 28896513100701
  name: Resetting Home Assistant Green failed. What can I do?
  position: 5
  labels: green, troubleshooting
---

## Symptom

> You followed the steps on [resetting the device without SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837) but the reset failed.

Reviewer: ```suggestion
You followed the steps on [resetting the device without SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837), but the reset failed.
```

> You followed the steps on [resetting the device without SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837), but the reset failed.

Author: ```suggestion
You followed the steps on [resetting the device without SD card (Operating System version 13.1 and later)](/hc/en-us/articles/25161225495837), but the reset failed.
```

## Resolution

> 1. Check which version of Home Assistant Operating System you have installed on Home Assistant Green.

Reviewer: I thought all of these articles were written with the assumption that the reader was running the current version?

Reviewer: ok.

   - Go to [**Settings** > **About**](https://my.home-assistant.io/redirect/info/).

> 2. If you have OS version 13.1 and later:

Reviewer: ```suggestion
2. If you have Home Assistant Operating System version 13.1 and later:
```

```suggestion
2. If you have Operating System version 13.1 and later:
```

Should be applied to more than just his line.

   - The cause might be that you did not wait long enough after shutting down the device. Repeat the steps on [resetting the device without SD card](/hc/en-us/articles/25161225495837) and after shut down, make sure to wait until all the LEDs turned off.
3. If you have OS version older than 13.1: Follow the steps on [resetting the device using an SD card (OS version 13.1 and older)](/hc/en-us/articles/25162566451485).

## Related topics

> - [Resetting the device using an SD card (OS version 13.1 and older)](/hc/en-us/articles/25162566451485)

Author: ```suggestion
- [Resetting the device using an SD card (Operating System version 13.1 and older)](/hc/en-us/articles/25162566451485)
```

> - [Resetting the device without SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837)

Reviewer: ```suggestion
- [Resetting the device without using an SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837)
```
Just to make it easier to spot the difference in the lines.
Maybe also add some emphasis on "without", "older", and "later")

> - [Resetting the device without using an SD card (OS version 13.1 and later)](/hc/en-us/articles/25161225495837)

Author: ```suggestion
- [Resetting the device without using an SD card (Operating System version 13.1 and later)](/hc/en-us/articles/25161225495837)
```
